generator/src/main/java/com/linkedin/pegasus/generator/DataSchemaParser.java

```diff
@@ -211,24 +209,33 @@ private void init(AbstractMultiFormatDataSchemaResolver resolver,
       return _schemaAndLocations;
     }
 
+    public Map<DataSchema, DataSchemaLocation> getBaseDataSchemaAndLocations()
+    {
+      return _schemaAndLocations.entrySet().stream().filter(entry -> !isExtensionSchemaLocation(entry))
+          .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+    }
+
     public Map<DataSchema, DataSchemaLocation> getExtensionDataSchemaAndLocations()
```

Comment:
Please add java doc on both these methods and also for getSchemaAndLocations. Given we have mutliple methods, it would be helpful to document their behavior.

Reply:
Added